tests/Utils/Validators/FooClosureValidator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I plan to switch to this in the near future, but want to keep the codebase consistent for now.

```suggestion
```